portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -127,18 +130,59 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
     ArrayList<Comment> comments = new ArrayList<Comment>();
     int numberToDisplay = getNumberToDisplay(request); 
+    String paginationInstruction = (String) request.getParameter("pageAction");
```

Comment:
Optional: 
consider defining all the request params somewhere so that someone reusing the code can have a rough idea of what's supported without reading the overall codebase

Reply:
I like this idea---I'm just not sure how to go about implementing it. Would I be defining the request parameters in a comment somewhere, or in a separate file, maybe?

Reply:
Yup, this could be just a file that lists a bunch of constants

Reply:
Okay, I'll add it to the list of things to do if I have extra time! (create an Entity file with the properties I'm using and create a file with the request parameters)